src/public/sass/ITCSS/06_components/pages/equipment/content/_equipment.scss

```
@@ -0,0 +1,89 @@
+@use '../../../../02_tools/breakpoints' as *;
+@use '../../../../02_tools/mixins' as *;
+@use '../../../../01_settings/colors' as *;
+@use '../../../../01_settings/fonts' as *;
+.equipment {
+  height: auto;
+  padding: 1.5rem;
+  border: 0.125rem solid $color-white;
+  padding-inline: 0.5rem;
+  text-decoration: none;
+  transition:
+    backdrop-filter 200ms ease-in-out,
+    border ease 500ms;
+
+  &--center {
+    grid-column: 2 / 3;
+  }
+
+  &__icon {
+    width: 4rem;
+    height: 4rem;
+    margin: 0 auto;
+    margin-bottom: 1rem;
+
+    @include phoneXL {
+      width: 11rem;
+      height: 9rem;
+    }
+
+    @include tablet {
+      width: 11rem;
+      height: 9rem;
+    }
+
+    @include desktop {
+      width: 24svh;
+      height: 15svh;
+    }
+
+    @include desktopXL {
+      width: 24svh;
+      height: 24svh;
+    }
```

Don't define again the height values, just use:
aspect-ratio property 

```suggestion
    width: 4rem;
    margin: 0 auto;
    margin-bottom: 1rem;
    aspect-ratio: 5 / 4;

    @include phoneXL {
      width: 11rem;
    }
    @include desktop {
      width: 24svh;
    }
```